### DISCUSSION TITLE
Example clarification

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#23] [Q&A] [answered]
On the CSV examples provided, lines 4-6 for the "Platform_Health_Insurance|PPO|", you have values in both the negotiated_dollar column as well as the negotiated_algorithm column.  I just want to confirm that is supposed to be one or the other and should not be both, is that accurate?

**Comment by RidenourK** [comment, marked as answer]
@JaSimps5 Thank you for your question. 

As indicated in the CY 2024 OPPS/ASC Final Rule, CMS anticipates that most if not all payer-specific negotiated charges will fall into one of three categories, depending on how a hospital has established them: (1) standard dollar amount, (2) standard algorithm or percentage, or (3) hybrid where a standard dollar amount can be identified but the final allowed amount is dependent on additional variables. An example of where we would expect to see a standard charge in dollars would be standard charges established under a fee schedule or where an identifiable dollar amount has been established for an item or service (for example, a per diem rate, a gross charge for an itemized item or service, or a cash discounted price for a service package). An example of a where we would expect to see a standard charge expressed as an algorithm would be when a hospital has negotiated a reimbursement for defined service packages (for example, hip replacement or colonoscopy) that are based on differential percentages of total billed (gross) charges (for example, 50 percent of total billed charges for hip replacement and 75 percent of total billed charges for colonoscopy). A hybrid would be a situation in which the hospital has established both a standard charge in dollars and there are additional variables that would modify the negotiated rate for a particular item or service. For example, a hospital may have established a payer-specific negotiated charge under the MS–DRG methodology where an adjusted base rate in dollars has been established for each DRG code, but the adjusted base rate may be further modified due to certain variable factors (such as outlier cases or transfers). 

In general, CMS recommends that each hospital, as a starting point, inspect their contracts with each third-party payer to identify all standard charges established as dollar amounts. Next, the hospital should populate, by payer and plan, the MRF with those standard charges in dollars and describe the item or service associated with each of the standard charges (along with any relevant billing or accounting codes). After that has been done, the hospital should identify whether the standard charge (in dollars) is subject to modification and what factors or variables (for example, algorithm) might cause the standard charge to change, and indicate those as instructed by the data dictionary for the particular format selected. If the hospital’s payer-specific negotiated charge is based on an algorithm within which no standard dollar amount can be determined, then the hospital should specify what percentage or algorithm determines the dollar amount for the item or service. CMS also finalized a requirement for hospitals to display an estimated allowed amount which would provide needed context, in dollars, for instances in which the hospital’s standard charge can only be expressed as a percentage or algorithm for a specified payer’s plan.

Please refer to [88 FR 82096](https://www.federalregister.gov/d/2023-24293/p-4970).
